src/antsibull_changelog/cli.py

```diff
@@ -448,15 +449,30 @@ def _get_pyproject_toml_version(project_toml_path: str) -> str | None:
     with open(project_toml_path, 'r', encoding='utf-8') as f:
         data = (toml if HAS_TOML else tomli).loads(f.read())
 
+    # PEP 621 project metadata (https://peps.python.org/pep-0621/#version)
+    if data.get('version'):
+        return data.get('version')
+
     tool_config = data.get('tool') or {}
 
+    # Python Poetry (https://python-poetry.org/docs/pyproject/#version)
     if 'poetry' in tool_config:
         poetry_config = tool_config['poetry']
         return poetry_config.get('version')
 
     return None
 
 
+def _get_package_json_version(package_json_path: str) -> str | None:
+    '''
+    Try to extract version from package.json.
+    '''
+    with open(package_json_path, 'rb') as f:
```

Comment:
I'd open this in text mode instead of bytes mode. I think that's what the `json` module prefers.

Reply:
Changed in e8054089d2fa2845f11f8b77b0003c69ecd3da55.